Lib/_py_abc.py

```diff
@@ -66,7 +67,7 @@ def register(cls, subclass):
             # This would create a cycle, which is bad for the algorithm below
             raise RuntimeError("Refusing to create an inheritance cycle")
         cls._abc_registry.add(subclass)
-        ABCMeta._abc_invalidation_counter += 1  # Invalidate negative cache
+        ABCMeta._abc_invalidation_counter += 1 # Invalidate negative cache
```

Comment:
```suggestion
        ABCMeta._abc_invalidation_counter += 1  # Invalidate negative cache
```

Please revert as it's purely cosmetic here (we also keep two spaces before a comment as per PEP-8)

Reply:
Reverted